Jenkinsfile

```diff
@@ -47,7 +47,8 @@ timestamps {
           } catch (ignore) {
             sh "cat nsp-report.txt"
             archiveArtifacts 'nsp-report.txt'
-            error "Node security check failed see the report for the errors"
+            notifyBuildResult channel: channel, color: 'warning',
+              message: 'Node security check failed see the report for the errors'
           }
           sh "rm nsp-report.txt"
         }
```

Comment:
do we want to update yarn.lock?

Reply:
I updated it to update our other dependencies, added it as a separate commit